src/backend/common/cache_clearing/get_affected_queries.py

```diff
@@ -345,5 +346,20 @@ def insight_updated(affected_refs: TAffectedReferences) -> List[TCacheKeyAndQuer
     for year in years:
         queries.append(insight_query.InsightsLeaderboardsYearQuery(year))
         queries.append(insight_query.InsightsNotablesYearQuery(year))
+        for district_abbreviation in RenamedDistricts.get_latest_codes():
+            queries.append(
+                insight_query.DistrictInsightQuery(
+                    Insight.INSIGHT_NAMES[Insight.DISTRICT_INSIGHTS_TEAM_DATA],
```

Comment:
Instead of hardcoding the insight name, maybe `name` should be in `_affected_references` as well?